azure-pipelines.yml

```diff
@@ -338,8 +338,8 @@ jobs:
       ConnectionStrings__CosmosDB: $(CosmosDB)
       AzureWebJobsEventHubSender: $(EventHub)
       AzureWebJobsEventHubReceiver: $(EventHub)
-      AzureWebJobsSecretStorageKeyVaultConnectionString: $(KeyVaultConnectionString)
-      AzureWebJobsSecretStorageKeyVaultName: $(KeyVaultName)
+      AzureWebJobsSecretStorageKeyVaultClientId: $(KeyVaultClientId)
+      AzureWebJobsSecretStorageKeyVaultUri: $(KeyVaultUri)
```

Comment:
I don't know where `KeyVaultName` is coming from here, is it a pipeline variable? This needs to also be updated to `KeyVaultUri`

Reply:
Found this in the KV, added